packages/components/src/components/textarea/textarea.ts

```diff
@@ -333,7 +333,7 @@ export default class SdTextarea extends SolidElement implements SolidFormControl
     }[this.size];
 
     return html`
-      <div part="form-control" class="text-left">
+      <div part="form-control" class="flex flex-col w-full text-left">
```

Comment:
Does this fully preserve the ability to set rows? When does it use rows, when the height? Some documentation or even better a story would be very helpful here.

Reply:
@yoezlem Anything you need here? :)